lms/djangoapps/commerce/models.py

```diff
@@ -15,7 +15,7 @@ class Meta(object):
 
     API_NAME = 'commerce'
     CACHE_KEY = 'commerce.api.data'
-    DEFAULT_RECEIPT_PAGE_URL = '/commerce/checkout/receipt/?orderNum='
+    DEFAULT_RECEIPT_PAGE_URL = '/checkout/receipt/?order_number='
```

Comment:
The `receipt_page` database field should go away entirely, and be replaced by this constant.